internal/host/validator.go

```diff
@@ -251,6 +251,10 @@ type validator struct {
 	providerRegistry registry.ProviderRegistry
 }
 
+func (v *validator) isMediaConnected(c *validationContext) ValidationStatus {
+	return boolValue(c.host.MediaStatus == nil || *c.host.MediaStatus != models.HostMediaStatusDisconnected)
```

Comment:
While MediaStatus is nil this function should return `ValidationPending` instead, rather than `ValidationSuccess`

Reply:
The host media status nil or disconnected is ok